### DISCUSSION TITLE
[QoL] [Request] Time format for comments

### DISCUSSION THREAD

**Discussion by pwn0r** [Docile-Alligator/Infinity-For-Reddit#925] [Ideas]
Thanks for this fantastic reddit client, I wasnt aware about it until recently though (only when it become available on fdroid). 

I found some options to reduce visual clutter and stuff, and now comments look *mostly* fantastic. 
For instance, toolbar is not shown by default and elapsed time is used instead of full timestamps.

You can notice that my suggestions have a lot in common in the way reddit is rendered on *desktop* browsers. Which is my preferred way actually. 

So basically it seems elapsed time is only shown as bulky **15 Minutes**, **5 Hours** and so on. And then a user is finished off by **Yesterday** :D I havent found an option to change specifically elapsed time format??

TLDR:

please format elapsed time like 

-  **15m**
-  **2h**
-  **1d**

**Comment by Docile-Alligator** [comment]
Oh this is a nice suggestion!
